makefiles/docker.inc.mk

```
@@ -104,6 +104,10 @@ DOCKER_MAKE_ARGS ?=
 # https://github.com/docker/for-mac/issues/2396
 ETC_LOCALTIME = $(realpath /etc/localtime)
 
+# Fetch the number of jobs from the MAKEFLAGS
+DOCKER_MAKE_JOBS = $(filter -j%,$(MAKEFLAGS))
```

```suggestion
DOCKER_MAKE_JOBS = $(filter -j -j%,$(MAKEFLAGS))
```

Is `-j` matched otherwise?

---

Yes it is:

```
koen@morgen ~/dev/RIOT-review $ BUILD_IN_DOCKER=1 make -C examples/default/ -j 
make: Entering directory '/home/koen/dev/RIOT-review/examples/default'
Launching build container using image "riot/riotbuild:latest".
docker run --rm --tty --user $(id -u) -v '/usr/share/zoneinfo/Europe/Amsterdam:/etc/localtime:ro' -v '/home/koen/dev/RIOT-review:/data/riotbuild/riotbase:delegated' -e 'RIOTBASE=/data/riotbuild/riotbase' -e 'CCACHE_BASEDIR=/data/riotbuild/riotbase' -e 'BUILD_DIR=/data/riotbuild/riotbase/build' -e 'RIOTPROJECT=/data/riotbuild/riotbase' -e 'RIOTCPU=/data/riotbuild/riotbase/cpu' -e 'RIOTBOARD=/data/riotbuild/riotbase/boards' -e 'RIOTMAKE=/data/riotbuild/riotbase/makefiles'     -v '/home/koen/dev/RIOT/.git:/home/koen/dev/RIOT/.git:delegated'   -w '/data/riotbuild/riotbase/examples/default/' 'riot/riotbuild:latest' make     -j 
Building application "default" for "native" with MCU "native".

"make" -C /data/riotbuild/riotbase/boards/native
```

---

`--jobs[=N]` is also translated to `-j[N]` by `make` before adding it to the `MAKEFLAGS`